### DISCUSSION TITLE
problem configuring ICS over UI

### DISCUSSION THREAD

**Discussion by christopherpross** [mampfes/hacs_waste_collection_schedule#2257] [Q&A]
Hey,

I try to setup the integration the first time, now over the new ui.
I have to use the ICS-module, because my city provides only a Ical file. You have to send a post request with some x-urlencoded form parameters.
Now I have the issue, that I can not enter the parameters for the formular values. I don't know which format is expected in the text box. In the docs, i found, that this is from type "dict". But the format: 
`{"strasse":"21","hausnr":"3","ical":"+icalendar"}`
is not working, the parameters looks like to parsed and not send correctly to the server.
Maybe someone can help me out at this point? How to enter the parameters and headers? Which format does expect the ui to work?

all the best,
Christopher

**Comment by 5ila5** [comment]
The parameters are treated as one string. I changed this in #2262. You can probably take them and URL encode them (if you see it in the network part of your browser you can change to view source (chromium) or raw (Firefox) to see how the parameters are actually transmitted.

You can probably use `strasse=21&hausnr=3&ical=%2Bicalendar` as parameters input

**Reply by christopherpross** [reply]
@5ila5  Thank you for the quick answer. I already tried to use the urlencoded string, in multible variations, including the one, that you mensioned. Unfornately this isn't work. I can easyly send this request with curl, hurl or python, so it have to be something with the parameter formating. 
Your Pull request is already merged, should I use the master version maybe, or do you have other ideas?

all the best,
Christopher
